snuba/datasets/configuration/issues/entities/search_issues.yaml

```diff
@@ -0,0 +1,101 @@
+version: v1
+kind: entity
+name: search_issues
+
+schema:
+  [
+    { name: organization_id, type: UInt, args: { size: 64 } },
+    { name: project_id, type: UInt, args: { size: 64 } },
+    { name: group_id, type: UInt, args: { size: 64 } },
+    { name: search_title, type: String },
+    { name: fingerprint, type: Array, args: { inner_type: { type: String } } },
+    { name: occurrence_id, type: UUID },
+    { name: occurrence_type_id, type: UInt, args: { size: 8 } },
```

Comment:
What is the ocurrence_type_id? In other documents related to this schema that I've seen, this was listed as a UInt64. Why did it change to UInt8?

Reply:
It's the type of the occurrence, it won't have many unique values. The values represent the specific issue type, so `n+1 database query`, `file io on main thread`, etc. I think it was a mistake that we listed it as a UInt64 in the spec doc.

Reply:
Ok

Reply:
Yeah, that was me not updating the doc. It should be UInt8.